portal-frontend/src/app/features/applications/edit-submission/proposal/incl-proposal/incl-proposal.component.html

```diff
@@ -192,13 +192,14 @@ <h3>Notification and Public Hearing Requirements</h3>
         <app-file-drag-drop
           id="advertising"
           [uploadedFiles]="noticeOfPublicHearing"
-          (uploadFiles)="attachFile($event, DOCUMENT.PROOF_OF_ADVERTISING)"
+          (uploadFiles)="attachProofOfAdvertising($event)"
```

Comment:
is this correct? noticeOfPublicHearing uses the attachProofOfAdvertising

Reply:
The title of the upload in the UI is Notice of Public Hearing but the backend stores it as a PROOF OF ADVERTISING.